rm-datasource/src/main/java/com/alibaba/fescar/rm/datasource/undo/mysql/keyword/MySQLKeywordChecker.java

```diff
@@ -524,24 +528,16 @@ public static KeywordChecker getInstance() {
          */
         public final String name;
 
-        MySQLKeyword() {
-            this(null);
-        }
-
         MySQLKeyword(String name) {
             this.name = name;
         }
     }
 
+
     @Override
     public boolean check(String fieldOrTableName) {
-        try {
-            if (StringUtils.isNotBlank(fieldOrTableName)) {
-                MySQLKeyword.valueOf(fieldOrTableName.toUpperCase());
-                return true;
-            }
-        } catch (IllegalArgumentException e) {
-            //do nothing
+        if (null != fieldOrTableName) {
```

Comment:
avoid extra toUpperCase  
better use KeyWord_Set contain first if contain return true;
how about the next code 

`
        if (KEYWORD_SET.contains(fieldOrTableName)) {
            return true;
        }
        if (null != fieldOrTableName) {
            fieldOrTableName=fieldOrTableName.toUpperCase();
        }
        return KEYWORD_SET.contains(fieldOrTableName);
`

Reply:
ok.I'll take your suggestion.thanks.